doc/api/cli.md

```diff
@@ -219,6 +219,15 @@ added: v2.1.0
 Prints a stack trace whenever synchronous I/O is detected after the first turn
 of the event loop.
 
+### `--async-hooks-sanity-check`
+<!-- YAML
+added: REPLACEME
+-->
+
+Enables sanity checks for async_hooks. These can also be enabled in runtime
+by enabling one of the `async_hooks` hooks.
```

Comment:
Is the intent to leave this flag in forever?
Also, would prefer to take `-sanity` out... just `--async-hooks-check`

Reply:
> Is the intent to leave this flag in forever?

Hopefully, we can become so robust the checks are always enabled.

> Also, would prefer to take `-sanity` out... just `--async-hooks-check`

That is fine :)